politeiawww/cmd/politeiawww_dbutil/README.md

```diff
@@ -1,35 +1,132 @@
 # politeiawww_dbutil
 
 politeiawww_dbutil is a tool that allows you to interact with the politeiawww
-database.
+database. 
+**Note**: The politeiawww database works with LevelDB and CockroachDB. 
+If you are using LevelDB you have to shut down politeiawww before using this tool.
 
-**Note**: You currently have to shut down politeiawww before using this tool.
 
+## Available Commands
+You can view the available commands and application options by using the help
+flag.
 
-## Usage
+`$ politeiawww_dbutil -h`
 
-You can specify the following options:
+You can view details about a specific command, including required arguments,
+by putting the help flag after the command.
 
+`$ politeiawww_dbutil <command> -h`
+
+
+## Setup Configuration File
+politeiawww_dbutil has a configuration file that you can setup to make execution
+easier. You should create the configuration file under the following paths.
+
+**macOS**
+
+```
+/Users/<username>/Library/Application Support/Politeiawww/dbutil/politeiawww_dbutil.conf
 ```
-    --testnet
-    Whether to interact with the testnet or mainnet database
 
-    --datadir <dir>
-    Specify a different directory where the database is stored
+**Windows**
 
-    --dump [email]
-    Print the contents of the entire database to the console, or the
-    contents of the user, if provided.
+```
+C:\Users\<username>\AppData\Local\Politeiawww/cli/politeiawww_dbutil.conf
+```
 
-    --setadmin <email> <true/false>
-    Sets or removes the given user as admin.
+**Ubuntu**
 
-    --addcredits <email> <quantity>
-    Adds proposal credits to the given user.
 ```
+~/.politeiawww/cli/politeiawww_dbutil.conf
+```
+
+Check the sample configuration file at `politeiawww_dbutil/sample-politeaiwww.conf`
+for more information.
+
+
+
+## Usage 
+
+
+### Migrating the database 
+
+If you are running an older version of the database you need to migrate
```

Comment:
I understand the need for a migration tool if you're migrating from leveldb to cockroachdb, but shouldn't version migrations be handled automatically inside of politeiawww?